### PR TITLE
pwgen: add page

### PR DESCRIPTION
**Pull request by sylvainbannier**
`pwgen` is a linux password generator. 
`pwgen` defaults generate pronouceable passwords. I think this would be usefull to provide quick access to options generating a more secure password.

### CODE REVIEW

File: pages/linux/pwgen.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# pwgen
+
+> Generate passwords.
+
+- Generate 8 characters long random passwords with symbols :
+
+`pwgen -sy`
```
Comment (ostera) [inline pages/linux/pwgen.md]: As per: http://linux.die.net/man/1/pwgen, you're missing the length (8). So a few samples could be:

```md
- Generate random password with symbols:

`pwgen -y {{length}}`

- Generate hard-to-memorize passwords:

`pwgen -s {{length}}`
```

File: pages/linux/pwgen.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# pwgen
+
+> Generate passwords.
```
Comment (ostera) [inline pages/linux/pwgen.md]: According to their webpage, they are a tool to `generate pronounceable passwords`

**Comment by rubenvereecken** [comment]
I tried the default and some of these are far from pronounceable, I feel cheated :(.

**Comment by ostera** [comment]
Thanks @sylvainbannier for your time! All the fixes are available here: https://github.com/tldr-pages/tldr/commit/dc3ba725f9b706d12a8baec971ac63fcb567080b

**Pull request merged**